File: calculator-assignment/Calculator.js
// Function to add two numbers
function add(num1, num2) {
  return num1 + num2;
}

// Function to subtract two numbers
function subtract(num1, num2) {
  return num1 - num2;
}

// Function to multiply two numbers
function multiply(num1, num2) {
  return num1 * num2;
}

// Function to divide two numbers
function divide(number1, number2) {
  return number1 / number2;
}

// Function to get input from the user
function getInput(message) {
  return parseFloat(prompt(message));
}

// Function to calculate the result based on the operator
function calculate() {
  const num1 = getInput("Enter the First number: ");
  const num2 = getInput("Enter the Second number: ");
  const operator = prompt("Enter the operator (+, -, *, /): ");

  let output;

  switch (operator) {
    case "+":
      output = add(number1, number2);
      break;
    case "-":
      output = subtract(number1, number2);
      break;
    case "*":
      output = multiply(number1, number2);
      break;
    case "/":
      output = divide(number1, number2);
      break;
    default:
      output = "Invalid operator";
  }

  alert("Result: " + output);
}

// Call the calculate function
calculate();

File: Coding Excerise/Coding Class/src/Calculator.java
import java.util.Scanner;

class Main {
    public static void main(String[] args ) {

        char operators;
        Double num1, num2, output;

        //creation of an object for scanner class
        Scanner input = new Scanner(System.in);

        // User entering the operator
        System.out.println("Enter the operator: ");
        operators = input.next().charAt(0);

        //user enters the First number
        System.out.println("Enter the First Number: ");
        num1 = input.nextDouble();

        //user enters the second number
        System.out.println("Enter the Second Number: ");
        num2 = input.nextDouble();

        switch (operators){

            //addition operation
            case '+':
                output = num1 + num2;
                System.out.println(num1 + "+" + num2 + "=" + output);
                break;

            // Subtraction Operation
            case '_':
                output = num1 - num2;
                System.out.println(num1 + "-" + num2 + "=" + output);
                break;

            //Multiplication Operation
            case '*':
                output = num1 * num2;
                System.out.println(num1 + "*" + num2 + "=" + output);
                break;

             //Division Operation
            case '/':
                output = num1 / num2;
                System.out.println(num1 + "/" + num2 + "=" + output);
                break;

            default:
                System.out.println( "Invalid Operator!");
                break;
        }

        input.close();
    }
}
